presto-orc/src/main/java/com/facebook/presto/orc/reader/StructStreamReader.java

```diff
@@ -93,28 +95,24 @@
     @Override
     public void setReferencedSubfields(List<SubfieldPath> subfields, int depth)
     {
-        HashMap<String, ArrayList<SubfieldPath>> subfieldPaths = new HashMap();
+        Map<String, List<SubfieldPath>> subfieldPaths = new HashMap();
```

Comment:
Can we rename this so that the key is included in the name?  i.e. what is the `String`?